docs/OpenTelemetry.md

```diff
@@ -0,0 +1,44 @@
+
+When an OpenTelemetry SDK instance is configured, the metrics will be exported and sent to the collector configured as part of your applications configuration. If you are not using OpenTelemetry, the metric functionality is a no-op and the events are never sent.
+
+In cases when metrics events are sent, they will not be viewable outside of infrastructure configured in your application, and are never available to the OpenFGA team or contributors.
+
+## Metrics
+
+### Supported Metrics
+
+| Metric Name                       | Type      | Description                                                                          |
+| --------------------------------- | --------- | ------------------------------------------------------------------------------------ |
+| `fga-client.request.duration`     | Histogram | The total request time for FGA requests                                              |
+| `fga-client.query.duration`       | Histogram | The amount of time the FGA server took to internally process nd evaluate the request |
+| ` fga-client.credentials.request` | Counter   | The total number of times a new token was requested when using ClientCredentials     |
+
+### Supported attributes
+
+| Attribute Name                 | Type     | Description                                                                                                             |
+| ------------------------------ | -------- | ----------------------------------------------------------------------------------------------------------------------- |
+| `fga-client.response.model_id` | `string` | The authorization model ID that the FGA server used                                                                     |
+| `fga-client.request.method`    | `string` | The FGA method/action that was performed (e.g. `check`, `listObjects`, ...) in camelCase                                |
+| `fga-client.request.store_id`  | `string` | The store ID that was sent as part of the request                                                                       |
+| `fga-client.request.model_id`  | `string` | The authorization model ID that was sent as part of the request, if any                                                 |
+| `fga-client.request.client_id` | `string` | The client ID associated with the request, if any                                                                       |
+| `fga-client.user`              | `string` | The user that is associated with the action of the request for check and list objects                                   |
+| `fga-client.request.retries`   | `int`    | The number of retries attempted (starting from 1 for the original request). Deprecated, use `http.request.resend_count` |
+| `http.request.resend_count`    | `int`    | The number of retries attempted (starting from 1 for the original request)                                              |
```

Comment:
We made a change on .NET so that this value starts from 0 as recommended by the OTEL spec, can you change it here?